<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1267</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Myanmar Text&quot;;</string>
   </property>
   <property name="text">
    <string>Consequuntur? Et Veritatis Velit Nesciunt Odio!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>431</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 81 32pt &quot;Tajawal ExtraBold&quot;;</string>
   </property>
   <property name="text">
    <string>THE MORNING</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Myanmar Text&quot;;</string>
   </property>
   <property name="text">
    <string>Larem Ipsum Dolor Sit Amet Consectetur Adipisiaing  Elit, Sit,</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tajawal ExtraBold</family>
     <pointsize>32</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 81 32pt &quot;Tajawal ExtraBold&quot;;</string>
   </property>
   <property name="text">
    <string>FRESH COFFEE IN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>1271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(159, 159, 159);</string>
   </property>
   <property name="text">
    <string>____________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1271</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;Noto Serif&quot;;
background-color: rgb(203, 159, 111);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../image/coffee5.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="About_Button">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton {
               width: 100%;
            padding: 10px;
            border: none;
			color: rgb(255, 255, 255);
			background-color: rgb(232, 174, 133);
            border-radius: 5px;
			font: 81 16pt &quot;Tajawal ExtraBold&quot;;
            }
QPushButton:hover {
background-color: rgb(190, 141, 109);

            }
</string>
   </property>
   <property name="text">
    <string>Get It Now</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CartButton">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>20</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: none;
color:none;
border:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../image/shopping-cart.png</normaloff>../image/shopping-cart.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="HomeButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: none;
border:none;
font: 14pt &quot;MS Shell Dlg 2&quot;;
</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChatButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: none;
border:none;
font: 14pt &quot;MS Shell Dlg 2&quot;;
</string>
   </property>
   <property name="text">
    <string>Chat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MenuButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: none;
border:none;
font: 14pt &quot;MS Shell Dlg 2&quot;;
</string>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AboutButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: none;
border:none;
font: 14pt &quot;MS Shell Dlg 2&quot;;
</string>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../image/٢٠٢٥٠١٣١_١٦٢٦٥٤-removebg-preview.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: none;
border:none;
font: 14pt &quot;MS Shell Dlg 2&quot;;
</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_6</zorder>
  <zorder>About_Button</zorder>
  <zorder>label_2</zorder>
  <zorder>label_5</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>CartButton</zorder>
  <zorder>HomeButton</zorder>
  <zorder>ChatButton</zorder>
  <zorder>MenuButton</zorder>
  <zorder>AboutButton</zorder>
  <zorder>label_7</zorder>
  <zorder>ExitButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
